ilastik/shell/gui/ui/ilastikShell.ui

```diff
@@ -88,7 +88,16 @@
              <property name="spacing">
               <number>0</number>
              </property>
-             <property name="margin">
```

Comment:
always beautiful to see the noise QtDesigner introduces ;)

Reply:
I've given up on manually correcting elements from `.ui` files...